src/commands/send_metric/set.rs

```diff
@@ -0,0 +1,26 @@
+use super::common_args::CommonMetricArgs;
+use crate::{
+    api::envelopes_api::EnvelopesApi,
+    utils::metrics::{
+        arg_parsers, normalized_payload::NormalizedPayload, types::MetricType, values::MetricValue,
+    },
+};
+use anyhow::Result;
+use clap::{command, Args};
+use sentry::protocol::EnvelopeItem;
+
+#[derive(Args)]
+pub(super) struct SetArgs {
```

Comment:
```suggestion
pub(super) struct SetMetricArgs {
```

I think this name is slightly clearer